backend/akr/src/main/java/fi/oph/akr/scheduled/StatisticsGeneratorScheduled.java

```diff
@@ -20,9 +20,9 @@ public class StatisticsGeneratorScheduled {
   @Resource
   private StatisticsService statisticsService;
 
-  @Scheduled(cron = "0 0 4 30 * *")
+  @Scheduled(cron = "0 29 8 * * *")
```

Comment:
Taitaa vieläkin olla väärin :)
`minute hour day-of-month month day-of-week` => `29 8 * * *`?

Reply:
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/annotation/Scheduled.html#cron--

Reply:
Ok, eli siis vissiin hieman eri formaatti. Tämä ok.